.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["env"]
+}
```

Comment:
Nit: add sublimetext single trailing newline package: https://stackoverflow.com/questions/37794699/sublime-text-3-ensure-only-one-trailing-newline-at-end-of-file